src/Routes/Routes.tsx

```diff
@@ -145,6 +154,13 @@ export const routes: Array<NestedRoute> = [
     label: 'navBar.transactionHistory',
     icon: <FaHistory />,
     main: TransactionHistory
+  },
+  {
+    path: '/flags',
+    label: 'Feature Flags',
+    icon: <FaFlag />,
+    hide: window.location.hostname === 'localhost' ? false : true,
```

Comment:
It's fine until you are using the app locally without any docker or vhost, but using a `.env` variable would be way more robust

Maybe webpack could provide the `production` or `development` value too

Wait for a maintainer advice before applying my review please 👍

Reply:
@lolboysg try using `getConfig().isDev()` here as a more robust check